### DISCUSSION TITLE
Error on Browsershot

### DISCUSSION THREAD

**Discussion by jhonnynho** [spatie/browsershot#725] [Bugs]
I am getting this error 
[2023-04-17 17:22:36] local.ERROR: ErrorException: Undefined array key "selector" in //vendor/spatie/browsershot/src/Browsershot.php:911

This is my code
 Browsershot::url("https://www.google.com")
                ->setNodeBinary('PATH %~dp0;%PATH%;')
                ->setNpmBinary('/usr/local/npm')
                ->setNodeModulePath('/node_modules')
                ->save($path);
                
How could I solve this? I am not setting selector or like so.

**Comment by baharsah** [comment]
I have same problem here

![image](https://github.com/spatie/browsershot/assets/69349170/c77b5a85-ed54-4843-95c1-ede6220e9f4e)
![image](https://github.com/spatie/browsershot/assets/69349170/d7468fdb-1131-47a7-8a69-06df6ce3f6bd)
